registry/api/errcode/errors.go

```diff
@@ -97,25 +101,28 @@ type Error struct {
 
 var _ error = Error{}
 
-// ErrorCode returns the ID/Value of this Error
+// ErrorCode returns the ID/Value of this Error.
 func (e Error) ErrorCode() ErrorCode {
 	return e.Code
 }
 
 // Error returns a human readable representation of the error.
 func (e Error) Error() string {
-	return fmt.Sprintf("%s: %s",
```

Comment:
The error value should be `CHAIN_ONBUILD_DISALLOWED`, which will become `chain onbuild disallowed: Chaining ONBUILD via "ONBUILD ONBUILD" isn't allowed`. When this is printed, if we want tight control here, using the raw error may not be desired over resolving `Error.Message`.

A better approach would be something like this: `DOCKERFILE_INVALID_SYNTAX`, which will become `dockerfile invalid syntax: Chaining ONBUILD via "ONBUILD ONBUILD" isn't allowed`.

This is why we've typically allowed setting the message to whatever you want. It avoids overfitting the errors.